client/me/reauth-required/test/security-key-form.test.tsx

```diff
@@ -0,0 +1,80 @@
+/**
+ * @jest-environment jsdom
+ */
+import { render, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { SecurityKeyForm } from '../security-key-form';
+
+describe( 'SecurityKeyForm', () => {
+	const currentUserId = 123;
+	const translate = ( str: string ) => str;
+	const loginUserWithSecurityKey = jest.fn();
+	const fetch = jest.fn();
+	const onComplete = jest.fn();
+
+	const setup = () =>
+		render(
+			<SecurityKeyForm
+				twoStepAuthorization={ {
+					loginUserWithSecurityKey,
+					fetch,
+				} }
+				translate={ translate }
+				onComplete={ onComplete }
```

Comment:
Actually, can't we remove that prop now that it's no longer in use?

Reply:
> Actually, can't we remove that prop now that it's no longer in use?

Whoops, yes! For context, I copied these tests over from another branch I was working on before I had realized the `onComplete` prop was unused, hence why it continues to exist here.

Reply:
This will also hopefully be more obviously caught once I get a chance to migrate the component implementation to TypeScript.

Reply:
> Actually, can't we remove that prop now that it's no longer in use?

Removed in d09415d582f.